[chore]: update changelog with known issue (#13408)
open-telemetry/opentelemetry-collector: CHANGELOG.md

## v1.36.0/v0.130.0

### ❗ Known Issues ❗

> - Due to a [bug](https://github.com/open-telemetry/opentelemetry-go/issues/7039) in the prometheus exporter, if you are configuring your own prometheus exporter, the collector's internal metrics will be emitted with the unit included in the metric name. For example, the metric `otelcol_exporter_sent_spans__spans__total` instead of `otelcol_exporter_sent_spans_total`. The workaround is to manually configure `without_units: true` in your prometheus exporter config

Reviewer: ```suggestion
- Due to a [bug](https://github.com/open-telemetry/opentelemetry-go/issues/7039) in the prometheus exporter, if you are configuring a prometheus exporter, the collector's internal metrics will be emitted with an unexpected suffix in its name. For example, the metric `otelcol_exporter_sent_spans__spans__total` instead of `otelcol_exporter_sent_spans_total`. The workaround is to manually configure `without_units: true` in your prometheus exporter config
```
